docs/tasks/issuers/setup-acme/dns01/index.rst

```diff
@@ -64,6 +64,26 @@ Example usage::
 
 .. _supported-dns01-providers:
 
+Delegated Domains for DNS01
+===========================
+
+By default, cert-manager will not follow CNAME records pointing to subdomains.
+
+If granting cert-manager access to the root DNS zone is not desired, then the
+_acme-challenge.example.com subdomain can instead be delegated to some other,
+less privileged domain.
+This is done by adding an additional entry in `solvers`:
+
+.. code-block:: yaml
+
+   solvers:
+   - dns01:
+       # Valid values are None and Follow
+       cnameStrategy: Follow
```

Comment:
Maybe just have `cloudflare: ...` underneath this line or something? To make it clear that this field is *in addition to*, and not *instead of* regular DNS config

Reply:
I used `clouddns` instead, to keep it consistent with the main example.